extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcTenantConfig.java

```diff
@@ -1,10 +1,7 @@
 package io.quarkus.oidc.runtime;
 
 import java.time.Duration;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
+import java.util.*;
```

Comment:
Also we don't use star imports. Can you double-check if your IDE is using the Quarkus formatter?

Reply:
Indeed. It was an IDE issue.